app/controllers/tracks/approaches_controller.rb

```diff
@@ -23,9 +23,11 @@ def show
     @approach = OpenStruct.new(
       author: OpenStruct.new(
         avatar_url: "https://avatars.githubusercontent.com/u/135246?v=4",
-        handle: "Erik",
+        handle: "ErikSchierboom",
+        name: "Erik Schierboom",
         formatted_reputation: 999
       ),
+      content_html: "<h1>Hello</h1>",
       title: "Utilising In-built Date Functions for Consistency",
       published_at: DateTime.now
     )
```

Comment:
@ErikSchierboom I presume this shouldn't be hardcoded? :)

Reply:
this was me :) I just used it as dummy data to show something.